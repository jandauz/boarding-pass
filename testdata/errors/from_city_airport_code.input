M1DESMARAIS/LUC       EABC123 1ULFRAAC 0834 326J001A0025 100