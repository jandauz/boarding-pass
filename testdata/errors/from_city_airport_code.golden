bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 1ULFRAAC 0834 326J001A0025 100"
  |                                ^ got "1UL"
  |
  = reason: data for "From City Airport Code" must be 3 alpha characters
